sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -198,3 +200,53 @@ case class TypeOf(child: Expression) extends UnaryExpression {
     defineCodeGen(ctx, ev, _ => s"""UTF8String.fromString(${child.dataType.catalogString})""")
   }
 }
+
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """
+    _FUNC_(expr) - Evaluate an expression and handle certain types of execution errors by returning NULL.
+    In cases where it is preferable that queries produce NULL instead of failing when corrupt or invalid data is encountered, the TRY function may be useful, especially when ANSI mode is on and the users need null-tolerant on certain columns or outputs.
+    AnalysisExceptions will not be handled by this, typically errors handled by TRY function are:
+
+      * Division by zero,
+      * Invalid casting,
+      * Numeric value out of range,
+      * e.t.c
```

Comment:
Probably, I think we need to define a set of exceptions we handle in this function, e.g., `ArithmeticException` and `RuntimeException`.